File: app/src/main/java/net/skyscanner/backpack/demo/stories/PanelFragment.java
package net.skyscanner.backpack.demo.stories;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import net.skyscanner.backpack.demo.ComponentDetailFragment;
import net.skyscanner.backpack.demo.R;

public class PanelFragment extends ComponentDetailFragment {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_panel, container, false);
    }

}

File: app/src/main/java/net/skyscanner/backpack/demo/stories/PanelFragment.kt
package net.skyscanner.backpack.demo.stories

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import net.skyscanner.backpack.demo.ComponentDetailFragment
import net.skyscanner.backpack.demo.R

class PanelFragment : ComponentDetailFragment() {

  override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
    super.merge(R.layout.fragment_panel)
  }

}
